src/v/config/configuration.cc

```diff
@@ -1811,6 +1811,14 @@ configuration::configuration()
       "violations. Normally, this options should be disabled.",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       false)
+  , cloud_storage_disable_metadata_consistency_checks(
+      *this,
+      "cloud_storage_disable_metadata_consistency_checks",
+      "Disable all metadata consistency checks. This will allow redpanda to "
+      "replay logs with inconsistent tiered-storage metadata. Normally, this "
+      "option should be disabled.",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      false)
```

Comment:
Should the default value be true for now? Otherwise, we can still run into the same issue. We will have the scrubber in 23.3, so we could selectively enable this for clusters where the scrub comes out clean.

Reply:
fixed